cmd/otelcorecol/go.mod

```
@@ -4,7 +4,7 @@ module go.opentelemetry.io/collector/cmd/otelcorecol
 
 go 1.21.0
 
-toolchain go1.21.12
+toolchain go1.21.11
```

```suggestion
toolchain go1.21.12
```
I assume this is unrelated to this PR and we should remove this change?

---

yes that's a good assumption